.github/workflows/build.yaml

```diff
@@ -15,13 +15,46 @@ jobs:
     runs-on: ${{ matrix.os }}
     steps:
       - uses: actions/checkout@v3
+      - name: Identify week number
+        shell: bash
+        run: |
+          date +'%V' > week-number.txt
+          date +'week-number=%V' >> $GITHUB_ENV
```

Comment:
> Is it tied to the scheduled runs?

Yes, excellent observation! `%U` rolls the week numbers on Sundays. `%V` rolls the week numbers on Mondays. Since this codebase has a cron job for Mondays, I aligned with that and used `%V`.

The difference between repos exists because when we first discussed this, `%U` was the first solve and I didn't think anything of it until I started working on this repo.